.github/workflows/continuous-integration.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v3
         with:
-          node-version: 16.x
+          node-version: 18.x
           cache: yarn
 
       # ignore scripts as indy-sdk will try to build and we don't have libindy
```

Comment:
This can now be deleted in the CI scripts?

Reply:
Not yet as indy-sdk is still included for 0.3/0.4 snippets